R/get.R

```diff
@@ -20,11 +20,13 @@
 #' @export
 getItems <- function(dgeObj, itemNames){
 
-    assert_that(!missing(dgeObj),
-                !missing(itemNames),
-                "DGEobj" %in% class(dgeObj),
-                any(c("character", "list") %in% class(itemNames))
-    )
+    assertthat::assert_that(!missing(dgeObj),
+                            !missing(itemNames),
+                            msg = "Specify both a DGEobj and at least one itemName to retrieve.")
+    assertthat::assert_that("DGEobj" %in% class(dgeObj),
+                            msg = "The DGEobj must be of class 'DGEobj'.")
+    assertthat::assert_that(any(c("character", "list") %in% class(itemNames)),
+                            msg = "Pass the itemNames as a single character string or a list of items to retrieve.")
```

Comment:
I'd reword this slightly to "...or a list of string names to retrieve".  Does it need to be a list or is a character vector also accepted?  If so we need to update the check and message.